CONTRIBUTING.md

```diff
@@ -71,13 +71,13 @@ for a minimal build of libcudf without using conda are also listed below.
 
 Compilers:
 
-* `gcc` version 9.3+
-* `nvcc` version 11.5+
-* `cmake` version 3.26.4+
+* `gcc` version 11.4+
+* `nvcc` version 11.8+
+* `cmake` version 3.29.6+
 
 CUDA/GPU:
 
-* CUDA 11.5+
+* CUDA 11.8+
```

Comment:
We still support older CUDA versions at runtime, [we test 11.4 in our nightlies](https://github.com/rapidsai/shared-workflows/blob/50e7362d6c18d7fefe3940e70ca00128ad36f894/.github/workflows/conda-cpp-tests.yaml#L77-L78). We only require 11.8 for builds.
```suggestion
* CUDA 11.4+
```

Reply:
Thanks, I realized that this was probably runtime after I pushed this up. Glad to get confirmation on that.
Should I change this to say `CUDA/GPU Runtime` perhaps? I think the driver version is still correct.

Reply:
Yes, you can clarify this is runtime information. Driver version is still correct as far as I know, but we have discussed removing driver information because it is implied by the compatible CUDA versions — and it would be one less piece of information to track and update.